installer/netdata-uninstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#shellcheck disable=SC2181
+
+# this script will uninstall netdata
+
+# Variables needed by script:
+#  - NETDATA_PREFIX
+#  - NETDATA_ADDED_TO_GROUPS
+
+if [ "$1" != "--force" ]; then
+	echo >&2 "This script will REMOVE netdata from your system."
+	echo >&2 "Run it again with --force to do it."
+	exit 1
+fi
+
+#shellcheck source=/dev/null
+source installer/.environment.sh || exit 1
+#shellcheck source=/dev/null
+source installer/functions.sh || exit 1
+
+echo >&2 "Stopping a possibly running netdata..."
+for p in $(pidof netdata); do run kill "$p"; done
```

Comment:
Since this scripts includes `functions.sh`, it would be safer to call `stop_all_netdata` (that one checks the namespace of the running netdata, so it won't stop netdata running in containers when uninstalling on the host).

Reply:
Please check this.

Reply:
done